src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -779,9 +769,9 @@ describe('ReactDOMServerIntegration', () => {
     });
 
     describe('unknown attributes', function() {
```

Comment:
I'd like to have a few more integration tests here. Similar to tests you added to `ReactDOMComponent-test`, I'd like to verify that server and client renderer behave consistently with regards to unknown attributes. We should assert that numbers/booleans/strings get passed through, objects/functions don’t, and that there warnings are emitted when necessary.

We can probably even delete such tests from `ReactDOMComponent-test` because these are more exhaustive (they test both client and server, and that they match up).